src/components/conditional-wrapper/ConditionalWrapper.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import * as React from 'react';
+
+/**
+ * conditional wrapper will disable the wrapper and only return the children if isDisabled is true
+ */
+type Props = {
+    children: React.Node,
+    isDisabled?: boolean,
+    wrapper: any,
+    wrapperProps?: Object,
+};
+
+const ConditionalWrapper = ({ children, isDisabled, wrapper, wrapperProps = {} }: Props) =>
```

Comment:
True, I think it's better to just modify the specific wrapper being used rather than introducing a component like this.

I removed the component and added a new prop `hasDynamicPosition` to `PopperComponent` instead, which optionally appends the computed style to the popper element.